mylang/private/maybe.bzl

```diff
@@ -0,0 +1,7 @@
+"maybe utilities"
```

Comment:
would there be more of them? This file doesn't feel worth the indirection to me, the two lines in internal_deps.bzl felt fine

Reply:
its a chicken and egg on this one; this little one liner is not something you can load from another rule set but rules almost always will end up with `internal_deps.bzl` and `dependencies.bzl` and it makes both of those much cleaner.

Reply:
```
load("@bazel_tools//tools/build_defs/repo:http.bzl", _http_archive = "http_archive")
load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")

def http_archive(name, **kwargs):
    maybe(_http_archive, name = name, **kwargs)
```
reduces to just one line
```
load("//mylang/private:maybe.bzl", http_archive = "maybe_http_archive")
```
in both those files and any other files a rule set has http_archives deps in

Reply:
the proper place would be in @bazel_tools but that may be a hard-one to upstream
```
load("@bazel_tools//tools/build_defs/repo:http.bzl", http_archive = "maybe_http_archive")
```

Reply:
removed from this PR; can do it as a follow-up if it is desirable